Lib/ufo2ft/filters/dottedCircleFilter.py

```diff
@@ -61,20 +108,34 @@ def __call__(self, font, glyphSet=None):
             glyphSet = _GlyphSet.from_layer(font)
 
         self.set_context(font, glyphSet)
-        dotted_circle = self.check_dotted_circle(glyphSet)
-        return self.check_dotted_circle_anchors(dotted_circle)
+        added_glyph = False
+        dotted_circle = self.check_dotted_circle()
+        if not dotted_circle:
```

Comment:
Random thought: an empty glyph will be False here I think. Maybe check for `is None`?

Reply:
I changed it to return the glyph name. If the name is "" you have bigger problems.

Reply:
but it doesn't matter here because it's a glyph name (str) not the glyph object